Documentation/nfs.md

```diff
@@ -38,6 +40,44 @@ rook-nfs-operator-879f5bf8b-gnwht       1/1     Running   0          29m
 rook-nfs-provisioner-65f4874c8f-kkz6b   1/1     Running   0          29m
 ```
 
+## Deploy NFS Admission Webhook (Optional)
+
+To enable admission webhook on NFS such as validating admission webhook, you need to do as following:
+
+First, ensure that `cert-manager` is installed.  If it is not installed yet, you can install it as described in the `cert-manager` [installation](https://cert-manager.io/docs/installation/kubernetes/) documentation. Alternatively, you can simply just run the single command below:
+
+```console
+kubectl apply --validate=false -f https://github.com/jetstack/cert-manager/releases/download/v0.15.1/cert-manager.yaml
+```
+
+This will easily get the latest version (`v0.15.1`) of `cert-manager`  installed. After that completes, make sure the cert-manager component deployed properly and is in the `Running` status:
+
+```console
+kubectl get -n cert-manager pod
+
+NAME                                      READY   STATUS    RESTARTS   AGE
+cert-manager-7747db9d88-jmw2f             1/1     Running   0          2m1s
+cert-manager-cainjector-87c85c6ff-dhtl8   1/1     Running   0          2m1s
+cert-manager-webhook-64dc9fff44-5g565     1/1     Running   0          2m1s
+```
+
+Once `cert-manager` is running, you can now deploy the NFS webhook:
+
+```console
+kubectl create -f webhook.yaml
+```
+
+Verify the webhook is up and running:
+
+```console
+kubectl -n rook-nfs-system get pod
+
+NAME                                    READY   STATUS    RESTARTS   AGE
+rook-nfs-operator-78d86bf969-k7lqp      1/1     Running   0          102s
+rook-nfs-provisioner-7b5ff479f6-688dm   1/1     Running   0          102s
+rook-nfs-webhook-74749cbd46-6jw2w       1/1     Running   0          102s
+```
```

Comment:
consider also updating the below section to include the new state column:

```
> kubectl -n rook-nfs get nfsservers.nfs.rook.io
NAME       AGE   STATE
rook-nfs   32s   Running
```

Reply:
Also add an instruction that explicitly tells the user to create the storage class file, e.g.:
```
> k apply -f sc.yaml
```

note an instruction is also needed that tells them to save it to a file called sc.yaml

Reply:
noted, i'll update it